src/main/resources/META-INF/resources/primefaces/menu/menu.menubutton.js

```diff
@@ -54,6 +75,14 @@ PrimeFaces.widget.MenuButton = PrimeFaces.widget.BaseWidget.extend({
      * @private
      */
     bindEvents: function() {
+        this.menuId = this.jqId + '_menu';
+        this.button = this.jq.children('button');
+        this.menu = this.jq.children('.ui-menu');
+        this.menuitems = this.jq.find('.ui-menuitem');
+        this.cfg.disabled = this.button.is(':disabled');
```

Comment:
Not sure why you moved this out of the `init` method?  Usually initialization of values like this is done in the `init` method.

Reply:
If initialization is in `init` method, `bindEvents` called from `TieredMenu#init` before initialization and raised script error 
caused by undefined reference.

But I found another solution. I'll fix this.
I'm sorry for my lack of experience with JavaScript.